docker-compose.yml

```
@@ -0,0 +1,64 @@
+services:
+  # PostgreSQL Database
+  postgres:
+    image: postgres:16-alpine
+    container_name: pc-postgres
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB:-pcdb}
+      POSTGRES_USER: ${POSTGRES_USER:-pcuser}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-pcpassword}
```

Use the env file here:
```suggestion
    env_file: ".env.example"
```
See:
https://docs.docker.com/compose/how-tos/environment-variables/set-environment-variables/